models/project/project.go

```diff
@@ -192,13 +294,13 @@ func (u *ProjectStruct) FetchGithub() (bool, error) {
 			issueTask.ListId = githubLabels[0].Id
 		} else {
 			// ついているlabelのlistを持ってない時
-			if *issue.State == "open" && openList != nil {
-				issueTask.ListId = openList.Id
+			if *issue.State == "open" && noneList != nil {
+				issueTask.ListId = noneList.Id
```

Comment:
__[gometalinter]__
* [gotype] invalid operation: noneList (variable of type *invalid type) has no field or method Id